airflow/operators/docker_operator.py

```diff
@@ -87,13 +90,15 @@ def __init__(
             volumes=None,
             xcom_push=False,
             xcom_all=False,
+            dockercfg_path=None,
+            username=None,
```

Comment:
Suggest renaming this to avoid confusion with the existing `user` argument. Maybe `registry_username`?